### PR TITLE
Allow players to spend 3x more time as an intern role (Technical Assistant, Medical Intern, Security Cadet)

### PR DESCRIPTION
**Pull request by JupiterFive**
## About the PR 
The intern roles are a nice and (mostly) stress-free introduction to their respective departments, but you can only spend 30 minutes as them, which is usually a single round. As a result beginners may not be comfortable enough to play an actual role in the department by the time they cannot play as it anymore; especially if the round was uneventful.

I made this pull request to try and solve this problem by allowing beginners more time as an intern.
EDIT: Forgot to clarify, originally I was going to change the maximum time to 90 minutes, but I decided to change it to 120 minutes instead shortly after.

**Changelog**

:cl: Jukereise
- tweak: Beginners can now spend up to 2 hours (90 extra minutes after unlocking the non-beginner jobs) as an intern (originally 30 minutes)

### CODE REVIEW

**Comment by asperger-sind** [comment]
Don't make PRs from your master, make a new branch and do your stuff there, then push it to upstream, just as a good habit.
Either way, LGTM, good first PR.

**Comment by JupiterFive** [comment]
I'll keep that in mind if I do make more contributions, thanks for the heads up.

**Comment by Illiux** [comment]
Why do we limit time spent in these roles anyway?

**Comment by asperger-sind** [comment]
> Why do we limit time spent in these roles anyway?

I mean, it just makes sense, if you are a player whom is experienced enough with a certain role, why would you play the trainee version of said role? Honestly, you could probably not even limit them, this restricts people who aren't familiar with the role enough to play the "I'm an intern, please don't judge me too hard" role after they've gone through "the bar", but what if you still don't feel like you're experienced enough, it's just annoying, besides, you can't even roleplay out internship with the intern job (e.g as an experienced player, some would probably like roleplaying as an intern once or twice for a neat scenario).

tl;dr because I ended up rambling, it kinda makes sense, but it kinda doesn't, with the current playerbase we could just remove the limiter entirely, because you can't roll antag and people generally want to roll antag, if not, they turn it off, on MRP people might want to roleplay as an intern, but can't because of said block.

**Comment by Illiux** [comment]
Yeah exactly. Many players wouldn't want play them given the option, but I don't really see what the reasoning would be for not giving the option. An experienced person might just want to roleplay it or have a low responsibility round, but not quite as low as a civilian role.

**Comment by moonheart08** [comment]
> Yeah exactly. Many players wouldn't want play them given the option, but I don't really see what the reasoning would be for not giving the option. An experienced person might just want to roleplay it or have a low responsibility round, but not quite as low as a civilian role.

Because some people continue to play them despite their limited slots and restrict actual newbies from trying to learn the game.

**Comment by asperger-sind** [comment]
> Because some people continue to play them despite their limited slots and restrict actual newbies from trying to learn the game. I added this limit after people who already unlocked jobs refusing to play anything else became an issue.

What if people with less playtime got prioritized for certain jobs, sorta like a weight system, that'd probably solve the issue.

**Comment by moonheart08** [comment]
> > Because some people continue to play them despite their limited slots and restrict actual newbies from trying to learn the game. I added this limit after people who already unlocked jobs refusing to play anything else became an issue.
> 
> What if people with less playtime got prioritized for certain jobs, sorta like a weight system, that'd probably solve the issue.

This would also require me to revisit the job selection algorithm and add *another* sorting factor.

**Comment by Illiux** [comment]
> revisit the job selection algorithm and add _another_ sorting factor.

Would this be worth a low-priority issue in case someone actually does feel like doing that?

**Comment by mirrorcult** [comment]
> > revisit the job selection algorithm and add _another_ sorting factor.
> 
> Would this be worth a low-priority issue in case someone actually does feel like doing that?

sure, i suppose

**Review by mirrorcult — APPROVED** [review]
this is 4x but semantics

**Comment by JupiterFive** [comment]
yeah, i forgot to change the title
